src/Itinero.IO.Osm/Itinero.IO.Osm.csproj

```diff
@@ -19,14 +19,12 @@
     <RepositoryUrl>https://github.com/itinero/routing/</RepositoryUrl>
     <RepositoryType>osm, openstreetmap, routing, mapping</RepositoryType>
   </PropertyGroup>
-  <ItemGroup>
-    <ProjectReference Include="..\Itinero\Itinero.csproj" />
-  </ItemGroup>
-  <ItemGroup>
-    <PackageReference Include="OsmSharp" Version="6.2.0" />
-  </ItemGroup>
   <ItemGroup>
     <Compile Include="..\..\SharedAssemblyVersion.cs" Link="Properties\SharedAssemblyVersion.cs" />
     <EmbeddedResource Include="Profiles\Defaults\*.lua" Exclude="bin\**;obj\**;**\*.xproj;packages\**;@(EmbeddedResource)" />
   </ItemGroup>
+  <ItemGroup>
+    <ProjectReference Include="..\..\..\osmsharp\core-develop\src\OsmSharp\OsmSharp.csproj" />
```

Comment:
This reference is local

Reply:
Yes sorry, I was testing against develop so I removed the nuget to ensure it wasn't something which was already fixed. You can likely remove the local reference and add the nuget back or I can do this when I get a second.

Reply:
Fixed in latest commit, changed back to nugets